clusterloader2/testing/watch-list/config.yaml

```diff
@@ -36,13 +36,13 @@ steps:
       Method: APIResponsivenessPrometheus
       Params:
         action: start
-- name: Start the secret informer
+- name: Start the secret informers
   phases:
     - namespaceRange:
         min: 1
         max: 1
         basename: watch-list
-      replicasPerNamespace: 1
+      replicasPerNamespace: 2
```

Comment:
This looks fine, but I think we probably need to first increase the cluster-size to have 2 nodes (I think 2 instances won't fit on a current single node, right?)

Reply:
I think we should be fine.

A single instance (`job`) of the performance test requires `6 CPUs`, `16 RAM`. The test runs on an `e2-standard-16` machine with `16 CPU, 64 GB`. It seems we should be able to run the second `job`.

Reply:
ok - let's see